forestry/api/lepidopterology/IButterflyMutation.java

```diff
@@ -5,10 +5,11 @@
  ******************************************************************************/
 package forestry.api.lepidopterology;
 
+import forestry.api.core.IClimateProvider;
 import forestry.api.genetics.IAllele;
 import forestry.api.genetics.IGenome;
 import forestry.api.genetics.IMutation;
 
 public interface IButterflyMutation extends IMutation {
-	float getChance(IButterflyNursery housing, IAllele allele0, IAllele allele1, IGenome genome0, IGenome genome1);
+	float getChance(IButterflyNursery housing, IAllele allele0, IAllele allele1, IGenome genome0, IGenome genome1, IClimateProvider climate);
```

Comment:
Rather than breaking backward compatibility with changing method signature. Why not overloading method with new argument and keep the original method without the `IClimateProvider`?

Reply:
This change was actually a mistake, and the file has been reverted. Sorry!